<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1083</width>
    <height>746</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: indigo;
</string>
   </property>
   <widget class="GLWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>701</width>
      <height>751</height>
     </rect>
    </property>
   </widget>
   <widget class="QTabWidget" name="tabWidget">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>90</y>
      <width>381</width>
      <height>321</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>General</string>
     </attribute>
     <widget class="QWidget" name="gridLayoutWidget_3">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>361</width>
        <height>80</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="0" column="2">
        <widget class="QRadioButton" name="radioButton_4">
         <property name="text">
          <string>Parallel</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="label_2">
         <property name="font">
          <font>
           <pointsize>15</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Projection</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QRadioButton" name="radioButton_5">
         <property name="text">
          <string>Central</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QDoubleSpinBox" name="doubleSpinBox">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>120</y>
        <width>171</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>20</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="minimum">
       <double>0.100000000000000</double>
      </property>
      <property name="maximum">
       <double>2.000000000000000</double>
      </property>
      <property name="singleStep">
       <double>0.010000000000000</double>
      </property>
      <property name="value">
       <double>0.500000000000000</double>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>130</y>
        <width>104</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>15</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Size</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Move</string>
     </attribute>
     <widget class="QWidget" name="gridLayoutWidget_2">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>-10</y>
        <width>304</width>
        <height>321</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="5" column="1">
        <widget class="QPushButton" name="rotateCounterclockwiseButton">
         <property name="text">
          <string>Counterclock</string>
         </property>
         <property name="shortcut">
          <string>Ctrl+9</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QPushButton" name="rotateUpButton">
         <property name="text">
          <string>Up</string>
         </property>
         <property name="shortcut">
          <string>Ctrl+8</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QPushButton" name="moveRightButton">
         <property name="text">
          <string>X--</string>
         </property>
         <property name="shortcut">
          <string>6</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QPushButton" name="moveLeftButton">
         <property name="text">
          <string>X++</string>
         </property>
         <property name="shortcut">
          <string>4</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QPushButton" name="rotateLeftButton">
         <property name="text">
          <string>Left</string>
         </property>
         <property name="shortcut">
          <string>Ctrl+4</string>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QPushButton" name="rotateRightButton">
         <property name="text">
          <string>Right</string>
         </property>
         <property name="shortcut">
          <string>Ctrl+6</string>
         </property>
        </widget>
       </item>
       <item row="5" column="0">
        <widget class="QPushButton" name="rotateClockwiseButton">
         <property name="text">
          <string>Clockwise</string>
         </property>
         <property name="shortcut">
          <string>Ctrl+1</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QPushButton" name="moveBackwardButton">
         <property name="text">
          <string>Z--</string>
         </property>
         <property name="shortcut">
          <string>9</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QPushButton" name="moveForwardButton">
         <property name="text">
          <string>Z++</string>
         </property>
         <property name="shortcut">
          <string>1</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QPushButton" name="moveUpButton">
         <property name="text">
          <string>Y++</string>
         </property>
         <property name="shortcut">
          <string>8</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QPushButton" name="rotateDownButton">
         <property name="text">
          <string>Down</string>
         </property>
         <property name="shortcut">
          <string>Ctrl+2</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QPushButton" name="moveDownButton">
         <property name="text">
          <string>Y--</string>
         </property>
         <property name="shortcut">
          <string>2</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>Look</string>
     </attribute>
     <widget class="QPushButton" name="changeBGColorButton">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>150</y>
        <width>141</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>background color</string>
      </property>
     </widget>
     <widget class="QWidget" name="gridLayoutWidget">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>30</y>
        <width>381</width>
        <height>90</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="1" column="4">
        <widget class="QRadioButton" name="radio_sphere">
         <property name="text">
          <string>sphere</string>
         </property>
         <property name="checked">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QCheckBox" name="checkBox_ln">
         <property name="text">
          <string>Lines</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QDoubleSpinBox" name="lnWidthSpinBox">
         <property name="focusPolicy">
          <enum>Qt::ClickFocus</enum>
         </property>
         <property name="decimals">
          <number>0</number>
         </property>
         <property name="minimum">
          <double>1.000000000000000</double>
         </property>
         <property name="maximum">
          <double>15.000000000000000</double>
         </property>
         <property name="singleStep">
          <double>1.000000000000000</double>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="QDoubleSpinBox" name="pnSizeSpinBox">
         <property name="focusPolicy">
          <enum>Qt::ClickFocus</enum>
         </property>
         <property name="decimals">
          <number>0</number>
         </property>
         <property name="minimum">
          <double>1.000000000000000</double>
         </property>
         <property name="maximum">
          <double>15.000000000000000</double>
         </property>
         <property name="singleStep">
          <double>1.000000000000000</double>
         </property>
         <property name="value">
          <double>1.000000000000000</double>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QPushButton" name="verticesColorButton">
         <property name="text">
          <string>Color</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QCheckBox" name="checkBox_pn">
         <property name="text">
          <string>Points</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QPushButton" name="edgesColorButton">
         <property name="text">
          <string>Color</string>
         </property>
        </widget>
       </item>
       <item row="1" column="3">
        <widget class="QRadioButton" name="radio_cube">
         <property name="text">
          <string>cube</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="0" column="3">
        <widget class="QCheckBox" name="checkBox_ln_2">
         <property name="text">
          <string>dotted</string>
         </property>
         <property name="checked">
          <bool>false</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_4">
     <attribute name="title">
      <string>Screen</string>
     </attribute>
     <widget class="QPushButton" name="screenshotButton">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>30</y>
        <width>143</width>
        <height>101</height>
       </rect>
      </property>
      <property name="text">
       <string>Screenshot</string>
      </property>
      <property name="shortcut">
       <string>P</string>
      </property>
     </widget>
     <widget class="QPushButton" name="screencastButton">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>140</y>
        <width>143</width>
        <height>101</height>
       </rect>
      </property>
      <property name="text">
       <string>Start Recording</string>
      </property>
      <property name="shortcut">
       <string>R</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QPushButton" name="QuitButton">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>460</y>
      <width>299</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Quit</string>
    </property>
    <property name="shortcut">
     <string>Q</string>
    </property>
   </widget>
   <widget class="QPushButton" name="loadModelFileButton">
    <property name="geometry">
     <rect>
      <x>900</x>
      <y>30</y>
      <width>171</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>OPEN</string>
    </property>
    <property name="shortcut">
     <string>Ctrl+O</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>10</y>
      <width>371</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>#FILENAME#</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="vectors">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>70</y>
      <width>161</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="vertex">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>40</y>
      <width>161</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>GLWidget</class>
   <extends>QWidget</extends>
   <header>glwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>QuitButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>848</x>
     <y>475</y>
    </hint>
    <hint type="destinationlabel">
     <x>877</x>
     <y>561</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
